frontend/app-development/features/appPublish/components/Deploy/DeployDropdown/DeployDropdown.test.tsx

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import { screen, waitFor, waitForElementToBeRemoved, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import type { DeployDropdownProps } from './DeployDropdown';
+import { DeployDropdown } from './DeployDropdown';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import type { ServicesContextProps } from 'app-shared/contexts/ServicesContext';
+import { renderWithProviders } from 'app-development/test/mocks';
+import type { AppRelease } from 'app-shared/types/AppRelease';
+import { BuildResult } from 'app-shared/types/Build';
+import { appRelease } from 'app-shared/mocks/mocks';
+
+const defaultProps: DeployDropdownProps = {
+  appDeployedVersion: '',
+  disabled: false,
+  setSelectedImageTag: jest.fn(),
+  selectedImageTag: 'test1',
+  startDeploy: jest.fn(),
+  isPending: false,
+};
+
+const created = '01.01.2024 18:53';
+
+const appReleases: AppRelease[] = [
+  {
+    ...appRelease,
+    tagName: 'test1',
+    created,
+    build: {
+      ...appRelease.build,
+      result: BuildResult.succeeded,
+    },
+  },
+  {
+    ...appRelease,
+    tagName: 'test2',
+    created,
+    build: {
+      ...appRelease.build,
+      result: BuildResult.succeeded,
+    },
+  },
+];
+
+const imageOptions = [
```

Comment:
Should we rename `image` to `release`? 🤔

Reply:
The type is named ImageOption tho? Should we still change it? Or change the type too? 🤔